interface/src/app/gene/searchFew/searchFew.js

```diff
@@ -76,6 +75,10 @@ angular.module('adage.gene.searchFew', [
         $scope.sendToNetwork = function() {
           CommonGeneFuncts.sendToNetwork($scope, $state);
         };
+
+        $scope.findEnrichedSignatures = function() {
+          CommonGeneFuncts.findEnrichedSignatures($scope, $state);
```

Comment:
Not really, because `searchFew.js` includes `adage.gene.selected` as a dependency, which includes `adage.gene.utils` as a dependency. See line 2 of `gene/selected/selected.js` file.

Reply:
Aha... that explains it then. It still feels weird to have this file depend upon `CommonGeneFuncts` without declaring that dependency, but if it's implicitly included already then we've technically got it covered.